mne/utils/docs.py

```diff
@@ -71,16 +71,19 @@ def _reflow_param_docstring(docstring, has_first_line=True, width=75):
 
 docdict['adjacency_clust'] = """
 adjacency : scipy.sparse.spmatrix | None | False
-    Defines adjacency between locations in the data, where "locations" can
-    be spatial vertices, frequency bins, etc. If ``False``, assumes no
-    adjacency (each location is treated as independent and unconnected).
+    Defines adjacency between locations in the data, where "locations" can be
+    spatial vertices, frequency bins, time points, etc. For spatial vertices,
+    see: :func:`mne.channels.find_ch_adjacency`. If ``False``, assumes
```

Comment:
That'd just give you a mass univariate and uncorrected t/F test of each "voxel" :shrug:

Reply:
exactly - so one could just use `stat_fun` on their data directly. But I get it now - using `adjacency=False` would implement the max-statistic permutation approach (without clusters). Might be worth saying that explicitly (as you see it took me a while to undestand why it might be useful :) ).

Reply:
but we can do it other time